src/internal.c

```diff
@@ -8829,7 +8973,7 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                         int ok;
 
                         store->error = ret;
-                        store->error_depth = args->totalCerts;
+                        store->error_depth = args->certIdx;
```

Comment:
This change looks okay assuming we document what `error_depth` is at the header level. Can you add that?